addons/payment/models/payment_acquirer.py

```diff
@@ -83,6 +83,7 @@ def _get_default_view_template_id(self):
         ('test', 'Test'),
         ('prod', 'Production')], string='Environment',
         default='test', oldname='env', required=True)
+    # TODO: move publish mixin in portal and use it on payment?
```

Comment:
I prefer to keep the mixins in website TBH, but if it was in portal this field could be removed and use the mixin, as payment depend on account which depend on portal
That would also mean overriding the mixin in website and only move minimal version in portal.
So I am not in favor or this, but wanted to point it

Reply:
:+1: